code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -314,7 +314,7 @@
 	New()
 		src.modules += new /obj/item/device/flashlight(src)
 		src.modules += new /obj/item/device/flash/cyborg(src)
-		src.modules += new /obj/item/weapon/pen/robopen(src)
+		src.modules += new /obj/item/weapon/pen/multi/robopen(src)

```

Comment:
I already mentioned that I was aware of that in the OP, it was just for the sake of addressing every grep result :p